assets/image-banner.css

```diff
@@ -155,6 +144,8 @@
 .image-banner__button-container {
   display: flex;
   align-items: center;
+  flex-wrap: wrap;
+  margin: 0 -8px;
```

Comment:
Can we avoid using a negative margin if we use `gap: 16px` to control the distance between elements?